misc/configs/nydusd-config.json

```diff
@@ -3,25 +3,23 @@
     "backend": {
       "type": "registry",
       "config": {
-        "scheme": "http",
         "timeout": 5,
         "connect_timeout": 5,
-        "retry_limit": 0
+        "retry_limit": 2
       }
     },
     "cache": {
-      "type": "blobcache",
-      "config": {
```

Comment:
why the `config` field get removed?

Reply:
@jiangliu After this [commit](https://github.com/containerd/nydus-snapshotter/commit/c5e1c57766aa4ac96fe80f7ec2c09223af72d845) in https://github.com/containerd/nydus-snapshotter/pull/229, the `work_dir` field is unnecessary.

Reply:
@jiangliu I misunderstood before, this field is still required when there is no snapshotter, it has been fixed now.